index.html

```diff
@@ -44,7 +44,7 @@
 <body>
   <frank-ly header="🚂🚃🚃💨"
       full-repo-names
-      repos='["notwaldorf/emoji-rain", "notwaldorf/emoji-selector", "polymer/polymer"]'
+      repos='["nzwulfin/vagrant-cluster", "projectatomic/atomic-site", "nzwulfin/contrib"]'
```

Comment:
Gah sorry, dirty branch!  I'll clean that up but probably not til Monday.
On Feb 12, 2016 6:10 PM, "Monica Dinculescu" <notifications@github.com>
wrote:

> In index.html
> <https://github.com/notwaldorf/frankly/pull/3#discussion_r52811119>:
>
> > @@ -44,7 +44,7 @@
> >  <body>
> >    <frank-ly header="🚂🚃🚃💨"
> >        full-repo-names
> > -      repos='["notwaldorf/emoji-rain", "notwaldorf/emoji-selector", "polymer/polymer"]'
> > +      repos='["nzwulfin/vagrant-cluster", "projectatomic/atomic-site", "nzwulfin/contrib"]'
>
> Would you mind reverting this one? I'm selling emoji! ✨💝🐼:)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/notwaldorf/frankly/pull/3/files#r52811119>.
>